worlds/shivers/__init__.py

```diff
@@ -0,0 +1,184 @@
+
+        #If front door option is on, determine which set of keys will be used for lobby access and add front door key to item pool
+        lobby_access_keys = 1
+        if self.options.front_door_usable:
+            lobby_access_keys = self.random.randint(1, 2)
+            itempool += [self.create_item("Key for Front Door")]
+        else:
+            itempool += [self.create_item("Heal")]
+
+        self.multiworld.itempool += itempool
+
+        #Lobby acess:
+        if self.options.lobby_access == 1:
+            if lobby_access_keys == 1:
+                self.multiworld.early_items[self.player]["Key for Underground Lake Room"] = 1
+                self.multiworld.early_items[self.player]["Key for Office Elevator"] = 1
+                self.multiworld.early_items[self.player]["Key for Office"] = 1
+            elif lobby_access_keys == 2:
+                self.multiworld.early_items[self.player]["Key for Front Door"] = 1
+        if self.options.lobby_access == 2:
+            if lobby_access_keys == 1:
+                self.multiworld.local_early_items[self.player]["Key for Underground Lake Room"] = 1
+                self.multiworld.local_early_items[self.player]["Key for Office Elevator"] = 1
+                self.multiworld.local_early_items[self.player]["Key for Office"] = 1
+            elif lobby_access_keys == 2:
+                self.multiworld.local_early_items[self.player]["Key for Front Door"] = 1
+
+    def pre_fill(self) -> None:
+        # Prefills event storage locations with duplicate pots
+        storagelocs = []
+        storageitems = []
+        self.storage_placements = []
+
+        for locations in Constants.location_info["locations_by_region"].values():
+            for loc_name in locations:
+                if loc_name.startswith("Accessible: "):
+                    storagelocs.append(self.multiworld.get_location(loc_name, self.player))
+
+        storageitems += [self.create_item(name) for name, data in item_table.items() if data.type == 'potduplicate']
+        storageitems += [self.create_item("Empty") for i in range(3)]
+
+        state = self.multiworld.get_all_state(True)
+
+        self.random.shuffle(storagelocs)
+        self.random.shuffle(storageitems)
+        
+        fill_restrictive(self.multiworld, state, storagelocs.copy(), storageitems, True, True)
+
+        self.storage_placements = {location.name: location.item.name for location in storagelocs}
+
+    set_rules = set_rules
+
+    def generate_basic(self):
+        self.multiworld.completion_condition[self.player] = lambda state: (Rules.first_nine_ixupi_capturable(state, self.player)
+                                                                           and Rules.lightning_capturable(state, self.player))
+
+    def _get_slot_data(self):
+        return {
+            'storageplacements': self.storage_placements,
```

Comment:
Minor nitpick, but most of the quotes earlier use `"`, but these are `'` quotes.

Reply:
Updated to use " instead of ' for consistency